### PR TITLE
Set CVO available condition to False

### PR DESCRIPTION
**Pull request by mresvanis**
This PR updates the `ClusterVersion`'s status condition of type `Available` to `status: "False"`, along with its `message: "Cluster version status unknown"`, in the OCP post-processing step of recert.

/cc @tsorya

### CODE REVIEW

File: src/ocp_postprocess.rs [hunk 1]
```diff
@@ -59,6 +59,8 @@ pub(crate) async fn ocp_postprocess(
 
     run_cluster_customizations(cluster_customizations, in_memory_etcd_client).await?;
 
+    set_cluster_version_available_false(in_memory_etcd_client).await?;
```
Comment (omertuc) [inline src/ocp_postprocess.rs:62]:
```suggestion
    // We do this because ...
    set_cluster_version_available_false(in_memory_etcd_client).await?;
```

**Review by omertuc — CHANGES_REQUESTED** [review]

File: src/ocp_postprocess.rs (hunk 1 above)
Comment (mresvanis) [reply src/ocp_postprocess.rs:62]: Good point, I'll talk with Igal to verify the reason and update this change with a proper comment.

**Comment by mresvanis** [comment]
/hold 

This change needs to be documented with the respective comment.

File: src/ocp_postprocess.rs (hunk 1 above)
Comment (tsorya) [reply src/ocp_postprocess.rs:62]: Currently it is very difficult to say when we finish installation and when data is not stale. 
Right now looking on cvo status does nothing as cvo doesn't even update this condition while starting in ibi/ibu.
This change allows us to monitor cluster installation with regular tools like looking on cvo available condition that will be set back to True when cvo will validate that everything is ok

Comment (mresvanis) [reply src/ocp_postprocess.rs:70]: @omertuc updated, PTAL.

**Pull request merged**